xbmc/addons/AddonManager.cpp

```diff
@@ -170,9 +167,12 @@ void CAddonMgr::FillCpluffMetadata(const cp_plugin_info_t* plugin, CAddonBuilder
     ADDONDEPS dependencies;
     for (unsigned int i = 0; i < plugin->num_imports; ++i)
     {
-      if (plugin->imports[i].plugin_id && plugin->imports[i].version)
-        dependencies.insert(std::make_pair(std::string(plugin->imports[i].plugin_id),
-            std::make_pair(AddonVersion(plugin->imports[i].version), plugin->imports[i].optional != 0)));
+      if (plugin->imports[i].plugin_id)
+      {
+        std::string id(plugin->imports[i].plugin_id);
+        AddonVersion version(plugin->imports[i].version ? plugin->imports[i].version : "0.0.0");
```

Comment:
To be clear: I don't see any technical problems with having version optional and implicitly defined as "0.0.0". We already do that for minversion, among other things